src/ontology/genepio-edit.owl

```diff
@@ -12644,6 +12705,89 @@ AnnotationAssertion(rdfs:label obo:GENEPIO_0100634 "State or Province or Territo
+SubClassOf(obo:GENEPIO_0100929 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100930 (soil sample probe)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100930 "A specimen collection device that is used to collect a soil sample from the ground."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100930 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100930 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100930 "2023-11-29T12:15:31Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100930 "soil sample probe")
+SubClassOf(obo:GENEPIO_0100930 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100932 (scooping)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100932 "A specimen collection process that is used to dig out and move an amount of sample using a scoop."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100932 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100932 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100932 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100932 "2023-11-29T11:21:54Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100932 "scooping")
+SubClassOf(obo:GENEPIO_0100932 obo:OBI_0600005)
+SubClassOf(obo:GENEPIO_0100932 obo:OBI_0600012)
+
+# Class: obo:GENEPIO_0100933 (sediment collection)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100933 "A specimen collection process that is used to collect sediment samples from the seabed with either a commercial diver or a grab device."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100933 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100933 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100933 "2023-11-29T11:40:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100933 "sediment collection")
+SubClassOf(obo:GENEPIO_0100933 obo:OBI_0600012)
+
+# Class: obo:GENEPIO_0100934 (soil coring)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100934 "A specimen collection process in which small-diameter boreholes are drilled into the ground to collect soil samples. Soil is then removed from boreholes in long clear tubes called soil cores."@en)
```

Comment:
Is it certain that "long clear tubes" are inserted into the boreholes to capture the soil cores - I don't think so.  "Soil cores" and "clear tubes" are a contradiction.  The soil core may be displayed in a long clear tube though.

Reply:
Resolved: Discussed to keep it simple
Definition: A specimen collection process that is used to collect soil sample cores.